phpstan-baseline.neon

```diff
@@ -3675,6 +3675,16 @@ parameters:
 			count: 1
 			path: lib/Extension/LanguageServerCompletion/Tests/Unit/Handler/HoverHandlerTest.php
 
+		-
+			message: "#^Cannot access property \\$result on Phpactor\\\\LanguageServer\\\\Core\\\\Rpc\\\\ResponseMessage\\|null\\.$#"
+			count: 1
+			path: lib/Extension/LanguageServerEvaluatableExpression/Tests/EvaluatableExpressionHandlerTest.php
+
+		-
+			message: "#^Parameter \\#1 \\$response of method Phpactor\\\\LanguageServer\\\\Test\\\\LanguageServerTester\\:\\:assertSuccess\\(\\) expects Phpactor\\\\LanguageServer\\\\Core\\\\Rpc\\\\ResponseMessage, Phpactor\\\\LanguageServer\\\\Core\\\\Rpc\\\\ResponseMessage\\|null given\\.$#"
+			count: 1
+			path: lib/Extension/LanguageServerEvaluatableExpression/Tests/EvaluatableExpressionHandlerTest.php
+
```

Comment:
the baseline is only allowed to get _smaller_ both of these cases should be handled. the fix for this is just:

```diff
diff --git a/lib/Extension/LanguageServerEvaluatableExpression/Tests/EvaluatableExpressionHandlerTest.php b/lib/Extension/LanguageServerEvaluatableExpression/Tests/Evaluata>
index f0d865ea..133801fd 100644
--- a/lib/Extension/LanguageServerEvaluatableExpression/Tests/EvaluatableExpressionHandlerTest.php
+++ b/lib/Extension/LanguageServerEvaluatableExpression/Tests/EvaluatableExpressionHandlerTest.php
@@ -30,6 +30,7 @@ class EvaluatableExpressionHandlerTest extends IntegrationTestCase
             'textDocument' => new TextDocumentIdentifier(self::PATH),
             'position' => PositionConverter::byteOffsetToPosition(ByteOffset::fromInt((int)$offset), $text)
         ]);
+        self::assertNotNull($response);
         $tester->assertSuccess($response);
         $result = $response->result;
         $this->assertInstanceOf(EvaluatableExpression::class, $result);
```

Reply:
Nice. Will clean up today as I get a chance. Thanks!